test/postgresql.autoupdate.test.js

```diff
@@ -477,6 +477,9 @@ describe('autoupdate', function() {
           'customerId': {
             'type': 'String',
             'length': 20,
+            'postgresql': {
```

Comment:
If the columnName is the same as property name, why do we have to specify it again here?

Reply:
We have to do that, if we don't the column name will be convert to lowercase. 
Please take a look at this issue. https://github.com/strongloop/loopback-connector-postgresql/issues/38

Reply:
I see thanks for pointing to the issue.